simulator/simple_planning_simulator/.idea/workspace.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hi @PhoebeWu21 
Just confirm. Did you add this file intentionally?

Reply:
Sorry, It was a accident. I have deleted it.